examples/js/loaders/GLTFLoader.js

```diff
@@ -1789,7 +1810,27 @@ THREE.GLTFLoader = ( function () {
 		var URL = window.URL || window.webkitURL;
 
 		var textureDef = json.textures[ textureIndex ];
-		var source = json.images[ textureDef.source ];
+
+		var textureExtensions = textureDef.extensions || {};
+
+		var source;
+
+		if ( textureExtensions[ EXTENSIONS.MSFT_TEXTURE_DDS ] ) {
+
+			if (!THREE.DDSLoader) {
+
+				throw new Error( 'THREE.GLTFLoader: Attempting to load .dds texture without importing THREE.DDSLoader' );
+
+			}
```

Comment:
This error would need to be thrown in the constructor of `GLTFTextureDDSExtension`, I think it will error out before it gets here otherwise.

Reply:
My bad, forgot to move it after I implemented GLTFTextureDDSExtention function. Fixed now :)